_data/chapters.yml

```diff
@@ -0,0 +1,55 @@
+# Table of Contents
+- id: intro
+  title: Introduction
+
+- id: concepts
+  title: Core Concepts
+  enabled: false
+  children:
+
+    - id: progressive-enhancement
+      title: Progressive Enhancement
+
+    - id: modularity
+      title: Modular Design
+
+    - id: web-standards
+      title: Web Standards
+
+    - id: style-guides
+      title: Style Guides
+
+    - id: frameworks
+      title: Frameworks
+
+- id: technical
+  title: Technical Guides
+  children:
+
+    - id: html
+      title: HTML
+      enabled: false
+
+    - id: css
+      title: CSS
+      children:
+
+        - id: css/architecture
+          title: CSS Architecture
+
+        - id: css/style-guide
+          title: Creating a Style Guide
```

Comment:
I don't understand why this is called "Creating a Style Guide". That sounds like a narrative on how you'd create your own style guide. I'd think this would be called "Style Guide?